e2e/publish_test.go

```diff
@@ -88,17 +92,22 @@ func testPublishVolume() {
 	It("should publish filesystem", func() {
 		mountTargetPath := "/mnt/csi-node-test"
 
+		nodeName := "topolvm-e2e-worker"
+		if isDaemonsetLvmdEnvSet() {
+			nodeName = getDaemonsetLvmdNodeName()
+		}
+
 		By("creating a logical volume resource")
-		lvYaml := []byte(`apiVersion: topolvm.cybozu.com/v1
+		lvYaml := []byte(fmt.Sprintf(`apiVersion: topolvm.cybozu.com/v1
```

Comment:
Could you replace it using embed.
https://golang.org/pkg/embed/

Reply:
I described our context.
As of go 1.16, it introduces embed feature, and we prefer using it instead of writing content directly to source code.
We tend to create `testdata` directory, put `some.yaml` in there, and use embed to read it.
https://github.com/cybozu-go/neco-apps/blob/main/test/storage_test.go#L21-L22

Reply:
BTW, this rewrite is not related to this PR and there are many candidates.
@llamerada-jp
I think the work should be separated into another PR.
How do you think?

Reply:
Sure. It is better to make another PR for replacing with embed.